modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/GranuleDescriptor.java

```diff
@@ -593,53 +597,6 @@ private void checkPamDataset() throws IOException {
         pamDataset = pamParser.parsePAM(auxFile);
     }
 
-    private boolean customizeReaderInitialization(ImageReader reader, Hints hints) {
```

Comment:
So this one moved in the InitializingReader extension point implementation? Nice. But the COG readers have no use for it?